### PR TITLE
3.1 correct stdout stderr in neo4j admin

### PR DESCRIPTION
**Pull request by benbc**

### CODE REVIEW

File: community/command-line/src/main/java/org/neo4j/commandline/admin/OutsideWorld.java [hunk 1]
```diff
@@ -19,7 +19,13 @@
  */
 package org.neo4j.commandline.admin;
 
-public interface Output
+public interface OutsideWorld
 {
-    void line( String text );
+    void stdOutLine( String text );
+
+    void stdErrLine( String text );
```
Comment (chrisvest) [inline community/command-line/src/main/java/org/neo4j/commandline/admin/OutsideWorld.java:26]: How about exposing `PrintStream`s instead?

**Comment by benbc** [comment]
@chrisvest Because `PrintStream` is a massive surface that we are unlikely to need most of and which is a huge pain to stub out. I'd be very happy to revisit this if we discover a need for more of `PrintStream`'s functionality, but it seems like unnecessary complexity at the moment.

**Comment by benbc** [comment]
Build failure was due to a breakage upstream. I've rebased this branch on top of the fix.

**Comment by benbc** [comment]
@srbaker Review, please.

**Comment by chrisvest** [comment]
How about passing `OutsideWorld` to `AdminCommand.execute` as well, or at least the IO parts of it? Commands need testing too.

**Comment by benbc** [comment]
@chrisvest Steven and I were wondering whether to do that. If you think it's useful then I will. Thanks.

**Comment by srbaker** [comment]
I like this. :+1:

**Pull request merged**